src/hooks/usePendingSwapData.ts

```diff
@@ -492,15 +496,15 @@ enum BridgePendingSwapTypes {
 async function fetchPendingSwapInfo(
   bridgeContract: Bridge,
   event: Event,
-  chainId: ChainId,
+  tokens: BasicTokens,
 ) {
   let result = null
   try {
     // this will throw if the itemId has already fully resolved
     // afik we don't have a better way of fetching current pendingSwaps
     // DEVELOPER: for this reason we don't use multicall
     const pendingSwap = event.args as unknown as BridgeEventGenericSwap
-    if (pendingSwap == null) return null
+    if (pendingSwap == null || !tokens) return null
```

Comment:
For the sake of keeping the scope of PR from pulling out TOKENS_MAP going to leave as is. It would be better to discuss conventions as a team and make one PR to replace everywhere.